hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestIncrementalReadWithFullTableScan.scala

```diff
@@ -69,14 +55,22 @@ class TestIncrementalReadWithFullTableScan extends HoodieSparkClientTestBase {
   }
 
   @AfterEach override def tearDown() = {
-    cleanupSparkContexts()
-    cleanupTestDataGenerator()
-    cleanupFileSystem()
+    spark = null
```

Comment:
shouldn't we set spark to null after cleaning up all resources?

Reply:
`spark` is a class variable that gets initialized before every test starts, should be fine to null out before cleaning up resources.